src/screens/ProfileScreen/index.js

```diff
@@ -218,6 +218,7 @@ const ProfileScreen = ({route}) => {
   const {mappingColorFeed} = useCoreFeed();
   const LIMIT_PROFILE_FEED = 10;
   const TYPE_GALLERY = 'gallery';
+  const refBottomSheet = React.useRef();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new ref `refBottomSheet` has been introduced but it seems like it's not being used anywhere in the provided hunks. If it's not used elsewhere in the code, consider removing it to avoid unnecessary memory allocation.
```diff
- const refBottomSheet = React.useRef();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->